ui/apps/ui/src/app/components/results-with-pagination/results-with-pagination.component.ts

```diff
@@ -37,29 +37,44 @@ import { Router } from '@angular/router';
       *ngIf="
         $any(results$ | async)?.length > 0 && (isLoading$ | async) === false
       "
-      ><ess-pagination
-        [paginationData]="$any(paginationData$ | async)"
-        [loading]="(isLoading$ | async) ?? false"
-        (activePageChange)="pageNr$.next($event)"
-      ></ess-pagination>
-      <ess-result
-        class="results"
+    >
+      <ng-container
```

Comment:
You accidentally cut top pagination, which was introduced earlier. Please put it back.

Reply:
yes, im sorry :). i restored